www/src/templates/template-docs-markdown.js

```diff
@@ -16,6 +16,26 @@ import Breadcrumb from "../components/docs-breadcrumb"
 import Container from "../components/container"
 import PrevAndNext from "../components/prev-and-next"
 
+import { useActiveHash } from "../hooks/use-active-hash"
+
+const getHeadingIds = (toc, traverseFullDepth = false) => {
+  const idList = []
+  const hashToId = str => str.slice(1)
+
+  if (toc) {
+    for (const item of toc) {
+      idList.push(hashToId(item.url))
+
+      // Only traverse sub-items if specified (they are not displayed in ToC)
+      if (`items` in item && traverseFullDepth) {
```

Comment:
`'items' in item` is also true if `item.items === null`
🤔 

in console:
```javascript
`index` in Object.create({index: null}) // ->true
```

Reply:
I realized that checking `item.items` is enough so I changed that in 1376382